### PR TITLE
Bios api

### PR DESCRIPTION
**Pull request by vierukundo**

### CODE REVIEW

File: src/server/controllers/UserController.js [hunk 1]
```diff
@@ -126,4 +126,24 @@ async function getUserfromToken(token) {
     return null;
 }
 
-module.exports = { registerUser, loginUser, tokenChecker };
\ No newline at end of file
+const getUserBio = async (req, res) => {
+    try {
+        const { userId } = req.body;
+        const usersCollection = await dbClient.getCollection('chatDB', 'users')
+        const user = await usersCollection.findOne({_id: userId});
```
Comment (NanorIsrael) [inline src/server/controllers/UserController.js:133]: I think searching with the raw string userId wont work convert it to an objectId

File: src/server/controllers/UserController.js [hunk 2]
```diff
@@ -126,4 +126,24 @@ async function getUserfromToken(token) {
...
+        const chats = await dbClient.getCollection("chatDB", "chats");
+
+        // gets all groups chats the user is member of
+        const chatGroups = chats.find({users: {$in: userId}, isRoomChat: true})
```
Comment (NanorIsrael) [inline src/server/controllers/UserController.js:137]: what do you do you expect chatGroups to be

**Review by NanorIsrael — CHANGES_REQUESTED** [review]
add some comments

File: src/server/controllers/UserController.js (hunk 1 above)
Comment (vierukundo) [reply src/server/controllers/UserController.js:133]: ooh, okay, i will change that

File: src/server/controllers/UserController.js (hunk 2 above)
Comment (vierukundo) [reply src/server/controllers/UserController.js:137]: It will be a list of all groups the user is part of

**Pull request merged**